### DISCUSSION TITLE
Invalid Content-Type für Genesenzertifikat

### DISCUSSION THREAD

**Discussion by msteffenDev** [Digitaler-Impfnachweis/certification-apis#73] [Q&A]
Folgender Fehler tritt auf:
Sorry, there is something invalid in your request: Invalid Content-Type (application/json) found for DocType (DGC_RECOVERY_CREATION)

Das Impzertifikat kann mit dem Content-Type "application/json" abgerufen werden !?

**Comment by dirkexe** [comment]
`application/vnd.dgc.v1+json` ist der geforderte Content-Type.

**Reply by msteffenDev** [reply]
Super, hat funktioniert.
Vielen Dank!
